pandas/tests/internals/test_internals.py

```diff
@@ -1250,14 +1250,20 @@ class TestCanHoldElement(object):
     ], ids=lambda x: x.__name__)
     def test_binop_other(self, op, value, dtype):
         skip = {(operator.add, 'bool'),
+                (operator.add, '<M8[ns]'),
                 (operator.sub, 'bool'),
                 (operator.mul, 'bool'),
+                (operator.mul, '<m8[ns]'),
+                (operator.mul, '<M8[ns]'),
                 (operator.truediv, 'bool'),
+                (operator.truediv, '<M8[ns]'),
                 (operator.mod, 'i8'),
                 (operator.mod, 'complex128'),
                 (operator.mod, '<M8[ns]'),
                 (operator.mod, '<m8[ns]'),
-                (operator.pow, 'bool')}
+                (operator.pow, 'bool'),
+                (operator.pow, '<m8[ns]'),
+                (operator.pow, '<M8[ns]')}
```

Comment:
These are good fixes.  Can you take it a step further and, instead of skipping, `pytest.raises(TypeError)` for the appropriate cases?